src/Linux/Avalonia.LinuxFramebuffer/Input/LibInput/LibInputBackendOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Avalonia.LinuxFramebuffer.Input.LibInput;
+
+/// <summary>
+/// LibInputBackend Options.
+/// </summary>
+public sealed record class LibInputBackendOptions
+{
+    /// <summary>
+    /// Used internally to pass libinput context to <see cref="LibInputBackend.InputThread(object?)"/>.
+    /// </summary>
+    internal IntPtr LibInputContext { get; init; }
+
+    /// <summary>
+    /// List Events of events handler to monitoring eg: /dev/eventX.
+    /// </summary>
+    public IReadOnlyList<string> Events { get; init; } = Array.Empty<string>();
```

Comment:
Should be nullable, so once we extend the class with extra options, one wouldn't have to manually scan devices and could just do `new LibInputBackendOptions()`

Reply:
I checked, it's nullable because Avalonia.LinuxFramebuffer doesn't have nullable enabled.

Reply:
I mean, it should be `null` by default, not empty. One shouldn't have to manually specify the list of input devices in normal usage